webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -88,38 +85,16 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
   sectionKey,
   title,
   children,
-  onResize,
   currentSize,
   menu
 }) => {
-  const [size, setSize] = useState<number>(currentSize || 400)
-
   const open = !sectionCollapsed
 
   useEffect(() => {
     window.dispatchEvent(new Event('resize'))
-  }, [size])
+  }, [currentSize])
 
-  const changeSize = (newSize: number) => {
-    if (size === newSize) {
-      return
-    }
-    sendMessage({
-      payload: { section: sectionKey, size: newSize },
-      type: MessageFromWebviewType.RESIZE_PLOTS
-    })
-    onResize(newSize)
-    setSize(newSize)
-  }
-  const menuItems: IconMenuItemProps[] = [
-    {
-      icon: Dots,
-      onClickNode: (
-        <SizePicker currentSize={size} setSelectedSize={changeSize} />
-      ),
-      tooltip: 'Resize'
-    }
-  ]
+  const menuItems: IconMenuItemProps[] = []
```

Comment:
[Q] Can we remove the menu altogether for template plots:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/203678360-809edb8c-b00c-4432-b919-048b1f9e302e.png">

Reply:
Same for images:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/203678548-677802fc-ae94-4815-b3b4-6bd125664a96.png">

What is the plan there? Are we dropping resizing from images altogether?

Reply:
> [Q] Can we remove the menu altogether for template plots:
> 
> <img alt="image" width="1728" src="https://user-images.githubusercontent.com/37993418/203678360-809edb8c-b00c-4432-b919-048b1f9e302e.png">

There was supposed to be a reset size button, but this PR was getting a little too big. I'll remove it until the new button it in place.

Reply:
> What is the plan there? Are we dropping resizing from images altogether?
The resizing will have to work differently as it's a table and stretches indefinitely. Right now resizing is more like changing the number of items per row from 1 to 4 and the comparison table is always one line. Resizing is disabled for now.